queries.sql

```diff
@@ -152,4 +152,8 @@ JOIN vets ON visits.vet_id = vets.id
 WHERE vets.name = 'Maisy Smith'
 GROUP BY species.name
 ORDER BY visit_count DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+SELECT COUNT(*) FROM visits where animal_id = 4;
+SELECT * FROM visits where vet_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
```

Comment:
- [ ] The screenshot that you attached for the query below depicting that you added the index to the `vet_id` column of the `visits` table does not show that the index was added through the sequential scan:

```sql
SELECT * FROM visits where vet_id = 2;
```

### The Screenshot

![image](https://github.com/abdozayan12/Vet-clinic/assets/77106962/3fdd5646-d96e-4af4-b17b-6cd27b0012be)

I would advise you to remove the index for this column and add it again. Please take screenshots of the `explain analyze` output one more time after removing and adding it again. 👍 😎

Reply:
I've removed the INDEX and recreated it
![Screenshot 2023-08-21 224720](https://github.com/abdozayan12/Vet-clinic/assets/114990830/f48bf3fe-9a2d-4928-bec5-ff2276b65a18)

![Screenshot 2023-08-21 223627](https://github.com/abdozayan12/Vet-clinic/assets/114990830/85d798ea-8366-4222-a328-991d81aaf6ea)
![Screenshot 2023-08-21 223653](https://github.com/abdozayan12/Vet-clinic/assets/114990830/545cfa50-2cb5-47da-a100-c6931607149c)

Reply:
@Gambit142